index.bs

```diff
@@ -1524,6 +1524,132 @@ the interaction with garbage collection.
 
 <h2 id="api-surface">JavaScript API Surface Concerns</h2>
 
+<h3>Choose the Appropriate WebIDL Construct for Data and Behavior (Dictionaries, Interfaces, and Namespaces)</h3>
+
+Web APIs commonly pass around data and functionality using WebIDL. As a specification author, decide carefully whether
```

Comment:
We generally go with something more direct for this first paragraph/sentence of each principle.